README.md

```diff
@@ -1,75 +1,96 @@
-
-For Icinga Web 2 we have to figure out if a system like `systemctl` is required, if a simple script can handle it, or if we don't need an init system at all.
-
-### Plugins
-We will make at least the official monitorig plugins available in the Icinga 2 containers. Users can extend the plugin set by using our Icinga 2 image as a base image and installing additional plugins on top. Additionally plugins can be made available through mount points.
-
-Another option that can be evaluated in the future is the possibility of having a separate image only for plugins. In this case Icinga 2 must be able to call those plugins remotely somehow. This is an early stage idea and would require furhter research.
-
-### Database Schema
-Our containers take care that the initial database schema is applied. On each start the container should check if an update is available and apply that as well.
-
-### Logs
-To make the logs available through the container engine we need to make sure to change the default logging output to stdout, respectively stderr.
-
-### Testing
-Testing must be implemented from the beginning. We'll have to research if there's any framework available that we can use. Otherwise we can use custom Makefiles in combination with docker-compose to test our images in different scenarios.
-
-### Build Platform
-This needs some further research to figure out what would work best for us. Available options include GitHub Actions, GitHub + TravisCI and GitLab CI/CD, but there may be other options as well.
-
-### Other Concerns
-Some other condcerns have been brought up that we should keep in mind. One is that long running containers (> 2 Weeks) may become stale and need a restart. It is unclear whether this is an Icinga issue or something else.
-
-Another concern is that networking can be a challenge when separating services into separate containers.
+# Icinga 2 - Docker image
+
+This image integrates [Icinga 2] into your [Docker] environment.
+
+## Usage
+
+```bash
+# CA
+docker run --rm \
+	-h icinga-master \
+	-v icinga-master:/data \
+	-e ICINGA_MASTER=1 \
+	icinga/icinga2 \
+	cat /var/lib/icinga2/certs/ca.crt > icinga-ca.crt
+
+# Ticket
+docker run --rm \
+	-h icinga-master \
+	-v icinga-master:/data \
+	-e ICINGA_MASTER=1 \
+	icinga/icinga2 \
+	icinga2 daemon -C
+docker run --rm \
+	-h icinga-master \
+	-v icinga-master:/data \
+	-e ICINGA_MASTER=1 \
+	icinga/icinga2 \
+	icinga2 pki ticket --cn icinga-agent > icinga-agent.ticket
+
+# Master
+docker run --rm -d \
+	--name icinga-master \
+	-h icinga-master \
+	-p 5665:5665 \
+	-v icinga-master:/data \
+	-e ICINGA_MASTER=1 \
+	icinga/icinga2
+
+# Agent
+docker run --rm -d \
+	-h icinga-agent \
+	-v icinga-agent:/data \
+	-e ICINGA_ZONE=icinga-agent \
+	-e ICINGA_ENDPOINT="icinga-master,$(docker inspect -f '{{.NetworkSettings.Networks.bridge.IPAddress}}' icinga-master),5665" \
```

Comment:
I would recommend using docker networks and DNS here. IP addresses can change after recreating containers.

Just create a new network:
```
docker network create icinga
```

Attach the containers by using `--network icinga` in `docker run` and they will be available via DNS (container name).